agent/newrelic-install.sh

```diff
@@ -120,7 +120,9 @@ error() {
 # Get the OS type.
 #
 ostype=
-if [ -f /etc/redhat-release -o -f /etc/redhat_version ]; then
+if [ -f /proc/version ] && [ -n "$(grep 'Alpine' /proc/version)" ]; then
+  ostype=alpine
+elif [ -f /etc/redhat-release -o -f /etc/redhat_version ]; then
   ostype=rhel
 elif [ -d /etc/dpkg ]; then
```

Comment:
How about this test to check if if's a Debian distribution:
```
  elif [ -f /etc/os-release ] && grep -q '^NAME="Debian.*' /etc/os-release; then
```
This was tested on all Debian releases:
```
for cn in jessie stretch buster bullseye; do echo "Testing on Debian $cn"; docker run --platform linux/amd64 debian:$cn grep '^NAME="Debian.*' /etc/os-release; done
Testing on Debian jessie
NAME="Debian GNU/Linux"
Testing on Debian stretch
NAME="Debian GNU/Linux"
Testing on Debian buster
NAME="Debian GNU/Linux"
Testing on Debian bullseye
NAME="Debian GNU/Linux"
```
This test is very strict and doesn't consider Ubuntu distribution to be a Debian distribution. If we want to roll up Ubuntu under Debian, the test can be relaxed to simply look for 'Debian' in /etc/os-release:
```
  elif [ -f /etc/os-release ] && grep -q  -i 'Debian' /etc/os-release; then
```

Reply:
Yes, please modify the check in 127 with the line that detects debian and remove the dpkg check!

`  elif [ -f /etc/os-release ] && grep -q  -i 'Debian' /etc/os-release; then`

thanks.

Reply:
Done [fb2273ea](https://github.com/newrelic/newrelic-php-agent/pull/391/commits/fb2273ea82f05eee4b94005adb9a41c1c8b5b2e1).